git_patrol_integration_test.py

```diff
@@ -125,6 +125,7 @@ def testTagAndBranchSuccess(self):
             'git', 'push', '--quiet', '--mirror', cwd=self._clone_dir))
     self.assertEqual(returncode, 0)
 
+    # Wait for Git Patrol to poll the new commits and record them.
     for i in range(3):
```

Comment:
I know this is not in scope for the change, but since you are updating the comment.
Retry 3 times? Why not more or less?  Magic number that could be named?

Reply:
Done.